### PR TITLE
Exiled.CustomRoles/API/Extensions::HasCustomrole

### PR DESCRIPTION
**Pull request by InziDeveloperMode**
-  Tested

### CODE REVIEW

File: Exiled.CustomRoles/API/Extensions.cs [hunk 1]
```diff
@@ -60,6 +60,25 @@ public static void Register(this IEnumerable<CustomRole> customRoles)
...
+        /// <returns><see langword="true"/> if the player has the custom role; otherwise, <see langword="false"/>.</returns>
+        public static bool HasCustomRole(this Player player, string roleName)
+        {
+            if (player is null)
+                throw new ArgumentNullException(nameof(player));
+
+            if (string.IsNullOrEmpty(roleName))
+                throw new ArgumentException("Role name cannot be null or empty.", nameof(roleName));
+
+            CustomRole customRole = CustomRole.Registered.FirstOrDefault(role => role.Name.Equals(roleName, StringComparison.OrdinalIgnoreCase));
```
Comment (VALERA771) [inline Exiled.CustomRoles/API/Extensions.cs:77]: Use CustomRole::TryGet

**Review by VALERA771 — COMMENTED** [review]
Also, cuz new custom roles API will be released soon, can be useless

**Comment by InziDeveloperMode** [comment]
> Also, cuz new custom roles API will be released soon, can be useless

but is there already this function in the new customrole API? Otherwise I might just switch branches.

**Comment by InziDeveloperMode** [comment]
Because if this function doesn't exist, it would be very useful to include it in exiled.

**Comment by VALERA771** [comment]
> Because if this function doesn't exist, it would be very useful to include it in exiled. 

If it's not there, switching branch won't help. This PR need to be remakes cuz APIs branch contains lots of changes

**Comment by InziDeveloperMode** [comment]
> > Because if this function doesn't exist, it would be very useful to include it in exiled. 
> 
> If it's not there, switching branch won't help. This PR need to be remakes cuz APIs branch contains lots of changes

I explained myself badly then, I meant that I will adapt the code to the new branch using the new customroles API

**Comment by InziDeveloperMode** [comment]
otherwise I will make a new pull request, as you prefer

**Pull request closed without merging**